.circleci/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM centos:7
+
+RUN yum update -y
+
+RUN  yum install -y \
+        https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+        git \
+        yum-utils \
+        centos-release-scl
+
+RUN yum install -y \
+        devtoolset-8 \
+        java-1.8.0-openjdk \
+        java-1.8.0-openjdk-devel \
+        patch \
+        python36 \
+        python36-devel
+
+RUN source /opt/rh/devtoolset-8/enable && \
+        python3.6 -m ensurepip && \
+        ln -s /usr/local/bin/pip3 /usr/bin/pip3 && \
+        pip3 install --upgrade pip setuptools && \
+        python3.6 -m venv /opt/tensorflow_venv
+
+RUN source /opt/rh/devtoolset-8/enable && \
+    source /opt/tensorflow_venv/bin/activate && \
+    pip3 install --upgrade \
+        'absl-py >= 0.1.6' \
```

Comment:
why not exact versions?

Reply:
The version expressiones here are taken from tensorflow's setup.py in the repo.